src/libmeasurement_kit/ooni/templates.cpp

```diff
@@ -76,12 +78,31 @@ void http_request(Var<Entry> entry, Settings settings, http::Headers headers,
         settings["http/method"] = "GET";
     }
 
+    /*
+     * XXX probe ip passed down the stack to allow us to scrub it from the
+     * entry; see issue #1110 for plans to make this better.
+     */
+    std::string probe_ip = settings.get("real_probe_ip_", std::string{});
```

Comment:
@hellais we initially discussed about renaming this `real_probe_ip` and that was sensible because the variable was passed down the stack. Since now is local and anyway the key passed down the stack is clear, I did not felt the need to also rename the variable (thus also allowing to stay within column 80).